src/components/Game/Game.js

```diff
@@ -56,6 +57,30 @@ class GameView extends Component {
         window.removeEventListener('resize', this.debouncedResizeListener);
     }
 
+    handleResize = () => {
+        const { letters, lettersInSlots, dragDisabled } = this.state;
+        if (isMobile() === dragDisabled) {
+            return;
+        }
+        if (!isMobile()) {
+            const lettersOutsideSlots = letters
+                .filter((letter) => letter.active);
+
+            this.setState({
+                letters: lettersInSlots.concat(lettersOutsideSlots),
+                currentAnswer: [...lettersInSlots.concat(lettersOutsideSlots)],
+                isSubmitVisible: false,
+            }, this.activateAllLetters);
```

Comment:
generally it's better (if convenient/possible) to calculate the new state and then pass it to `this.setState` once, I think it makes code easier to reason about and also UI has no chance to display intermediary state (between `setStates`) for a fraction of a second (no UI glitch)

here `setState` (line 69) triggers another `setState` (`activateAllLetters`) and the last statement calls function that also call `setState`, a lot happens here

I would suggest following structure:
```
let newState = { … }
<anything needed to compute newState>
this.setState(newState); // once and for good
```